### PR TITLE
improve profile.js test from 55% to 100%

### PR DESCRIPTION
**Pull request by juliemar**
### Description
This PR improves the test coverage of profile.js from 55% to 100%

### How to test?
run `npm test`

### Expected behavior
Code coverage of profile.js now is 100%
image

### CODE REVIEW

File: test/frontend/profile.test.js [hunk 1]
```diff
@@ -2,22 +2,74 @@ require("mock-local-storage");
...
-      name: "Nome do fulano",
-      imageUrl: "http://localhost/img.jpg",
-      email: "Mail@mail.com"
-    };
+  it("must return false if credentials is not present", () => {
+    const matrixProfile = new MatrixProfile();
+    assert.equal(matrixProfile.isProfileStored(), false);
+  });
+
+  it("must return true if credentials is not present", () => {
```
Comment (lccezinha) [inline test/frontend/profile.test.js]: I got confused with this description and the current assert, is that correct?

File: test/frontend/profile.test.js [hunk 2]
```diff
@@ -2,22 +2,74 @@ require("mock-local-storage");
...
+    matrixProfile.storeProfileData(profileData);
+
+    assert.equal(matrixProfile.userName(), profileData.name.split(" ").pop());
+  });
+
+  it("must return last user name null if profile data is not present", () => {
+    const matrixProfile = new MatrixProfile();
+    matrixProfile.terminate();
+
+    assert.equal(matrixProfile.userName(),null);
```
Comment (lccezinha) [inline test/frontend/profile.test.js]:
```suggestion
    assert.equal(matrixProfile.userName(), null);
```

File: test/frontend/profile.test.js (hunk 1 above)
Comment (juliemar) [reply test/frontend/profile.test.js]: yess I'm sorry CTRL+C CTRL+V :) the correct description is : must return true if user credentials is present"

Comment (lccezinha) [reply test/frontend/profile.test.js]: Now it makes sense!

**Pull request merged**